src/Api/DataTransformers/JobInputDataTransformer.php

```diff
@@ -64,7 +64,7 @@ private function setMinNotificationLevel (Job $job, $minNotificationLevel)
 
     private function setNotificationsEmail (Job $job, $notificationsEmail)
     {
-        if (isset($notificationsEmail) && !filter_var($notificationsEmail, FILTER_VALIDATE_EMAIL)) {
+        if (isset($notificationsEmail) && !filter_var($notificationsEmail, FILTER_VALIDATE_EMAIL) && !empty($notificationsEmail)) {
```

Comment:
This would allow the following values for $notificationsEmail:

    "" (una cadena vacía)
    0 (0 como un integer)
    0.0 (0 como un float)
    "0" (0 como un string)
    null
    false
    array() (un array vacío)

I think we only want to allow null and "", check especifically for those instead. I find it prefeable also to just allow null, and not to allow "", as the later is not retained and will be returned as null after setting it as "" (ate least it seemed to happen that in my tests...)